setup/gcp/installer/forseti_server_installer.py

```diff
@@ -92,36 +123,85 @@ def deploy(self, deployment_tpl_path, conf_file_path, bucket_name):
                 self.gcp_service_account,
                 self.user_can_grant_roles)
 
+            # Merge all the old rules if necessary
+            if self.migrate_from_v1:
+                self.merge_old_rules()
+
             # Copy the rule directory to the GCS bucket
             files.copy_file_to_destination(
                 constants.RULES_DIR_PATH, bucket_name,
                 is_directory=True, dry_run=self.config.dry_run)
 
+            self.print_copy_statement(constants.RULES_DIR_PATH, bucket_name)
+
+            # Waiting for VM to be initialized
             instance_name = '{}-vm'.format(deployment_name)
             self.wait_until_vm_initialized(instance_name)
 
-            # Create firewall rule to block out all the ingress traffic
-            gcloud.create_firewall_rule(
-                self.format_firewall_rule_name('forseti-server-deny-all'),
-                [self.gcp_service_account],
-                constants.FirewallRuleAction.DENY,
-                ['icmp', 'udp', 'tcp'],
-                constants.FirewallRuleDirection.INGRESS,
-                1)
-
-            # Create firewall rule to open only port tcp:50051
-            # within the internal network (ip-ranges - 10.128.0.0/9)
-            gcloud.create_firewall_rule(
-                self.format_firewall_rule_name('forseti-server-allow-grpc'),
-                [self.gcp_service_account],
-                constants.FirewallRuleAction.ALLOW,
-                ['tcp:50051'],
-                constants.FirewallRuleDirection.INGRESS,
-                0,
-                '10.128.0.0/9')
+            # Create firewall rules
+            self.create_firewall_rules()
 
         return success, deployment_name
 
+    def merge_old_rules(self):
+        """Merge old rules to new rules."""
+        for v1_rule in self.v1_config.rules:
+            new_rule_path = os.path.join(constants.RULES_DIR_PATH,
+                                         v1_rule.file_name)
+            new_rule = files.read_yaml_file_from_local(new_rule_path)
+            field_identifiers = {'rules': ['name', 'rule_id'],
+                                 'default_identifier': 'name',
+                                 'resource': 'type', 'bindings': 'role',
```

Comment:
Do you want to put ```'bindings': 'role',``` on its own line?  Easier to see it

Reply:
Updated.